wgpu-core/src/command/pass.rs

```diff
@@ -236,41 +236,41 @@ where
     Ok(())
 }
 
-pub(crate) fn set_push_constant<E, F: FnOnce(&[u32])>(
+pub(crate) fn set_immediate_data<E, F: FnOnce(&[u32])>(
     state: &mut PassState,
-    push_constant_data: &[u32],
+    immediate_data_data: &[u32],
```

Comment:
```suggestion
    immediate_data: &[u32],
```

`x_data_data` is used in a bunch of places, it should probably be `x_data`.